test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
@shabanovd Good idea to reference some local file that is corrupted.

A more roundabout, but attractive, approach would be to submit a test to qt3, and then have eXist run this test along with all the others. (We'd need to get the qt3 runner working again though...)